src/main/java/com/homework/lesson14/abstractfactory/AbstractFactory.java

```diff
@@ -0,0 +1,17 @@
+package com.homework.lesson14.abstractfactory;
+
+public class AbstractFactory {
+    private static final String uaFactory = "UA";
```

Comment:
Naming is not right.
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names